04-vector-raster-integration-advanced.Rmd

```diff
@@ -440,33 +470,34 @@ head(av_tree_height_tower)
 #how many pixels were extracted
 nrow(av_tree_height_tower)
 
-
 ```
 
 Notice that the command above extracted 1259 pixels from our LiDAR derived canopy
 height model. Does this number make sense?
 
+The mean tree height around the tower (22.39m) is similar to the mean height for 
+the whole tower area of interest (22.43m).  
+
 <div id="challenge" markdown="1">
 #Challenge: Extract Raster Height Values For Plot Locations
 
-Use the plot location points shapefile `HARV/plot.locations_HARV.shp` to 
-extract an average tree height value for each plot location in the study area.
+Use the plot location points shapefile `HARV/plot.locations_HARV.shp` or spatial
+object `plot.locationsSp_HARV` to extract an average tree height value for each
+plot location in the study area.
```

Comment:
Oops - we need to say to extract an average tree height value within a 20 m radius...